### PR TITLE
add date-box

### PR DESCRIPTION
**Pull request by kf-liu**
<p align="center">
image
  <h3 align="center">date-box</h3>
  <p align="center">🗓 Update a pinned gist to contain your anniversaries and countdown days (or hours)</p>
</p>

[Node.js CI](https://github.com/kf-liu/date-box/actions/workflows/node.yml)
[Update gist with Date](https://github.com/kf-liu/date-box/actions/workflows/schedule.yml)

---

> 📌 This project is inspired by [waka-box](https://github.com/matchai/waka-box) and [lang-box](https://github.com/inokawa/lang-box). 
> 
> 📌 For more pinned-gist projects like this one, check out: https://github.com/matchai/awesome-pinned-gists.

Thx!

### CODE REVIEW

**Review by ChrisCarini — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Displaying more detailed GitHub user data in a pinned gist.
 - [github-stats-box](https://github.com/bokub/github-stats-box) - Update a pinned gist to contain your GitHub statistics.
 - [lang-box](https://github.com/inokawa/lang-box) - Update a pinned gist to contain languages of your recent commits in GitHub
 
+## Custom
+
+Displaying data from user-defined content in a pinned gist.
```
Comment (ChrisCarini) [inline README.md:58]: Hi @kf-liu - what do you think about renaming this section to be `User Defined` so it's a bit more clear?

```suggestion
## User Defined

Displaying data from user-defined (e.g. configured in the GitHub Action input parameters) content in a pinned gist.
```

Comment (ChrisCarini) [reply README.md:58]: It would also be good to add this section in the `Contents` section at the top, as well.

Comment (kf-liu) [reply README.md:58]: Both are OK for me, thanks a lot!

Comment (ChrisCarini) [reply README.md:58]: cool! commit the suggestion, and I can merge in this PR for ya :)

**Comment by kf-liu** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您的邮件我已收到，谢谢！

**Pull request merged**